pyproject.toml

```diff
@@ -19,6 +19,9 @@ python-dotenv = "^1.0.0"
 pre-commit = "^3.5.0"
 parameterized = "^0.9.0"
 spacy = "^3.7.2"
+datasets = "^2.14.6"
+pillow = "^10.1.0"
+torch-utils = "^0.1.2"
```

Comment:
I think torch-utils will automatically try to import torch, which is not listed directly as a dependency.

Unfortunately Torch is not natively supported by Poetry, so we'll have to do some tricks like:

```
torch = [
    {url = "https://download.pytorch.org/whl/cpu/torch-1.13.1-cp310-none-macosx_11_0_arm64.whl", platform = "darwin"},
    {url = "https://download.pytorch.org/whl/cu117/torch-1.13.1%2Bcu117-cp310-cp310-linux_x86_64.whl", platform = "linux"}
]
```

(This is from a project that allows cross-platform use for linux/macos (darwin)) I'm using MacOS, so I think it would be best to add torch dependencies for all 3 platforms. Torch should be the only one with different build platforms (and maybe torchvision/torchaudio).

Reply:
This is why the Build pipeline has failed.